chia/cmds/wallet_funcs.py

```diff
@@ -280,7 +280,7 @@ async def get_offers(args: dict, wallet_client: WalletRpcClient, fingerprint: in
         records = [await wallet_client.get_offer(hexstr_to_bytes(id), file_contents)]
         if filepath is not None:
             with open(pathlib.Path(filepath), "w") as file:
-                file.write(records[0].offer.hex())
+                file.write(Offer.from_bytes(records[0].offer).compress().hex())
```

Comment:
We're compressing but then expanding 2x with `.hex()`?

Reply:
We want offers to be ASCII characters so that people can just open them and paste them somewhere if they want.  So while it's not the smallest file size, it's the smallest it's going to get for copy/paste which is what we care about.

Reply:
bech32m encodes binary smaller (and so does base64, but base64 doesn't copy/paste as nicely as bech32m). It's 5 bits/character, as opposed to 4 bits/character, and allows for a little human-readable header magic to hint at the type of the contents.

Reply:
Just sayin